File: c++/INTEST.cc
#include <iostream>

using namespace std;

int main(){
    int a,b,c;
    int result = 0;
    cin >> a;
    cin >> b;
    while (a>0)
    {
        cin >> c;
        if (c%b==0)
        {
            result++;
        }
        
        a--;
    }

    cout << result << endl;

    return 0;
}

File: java/INTEST.java
import java.util.Scanner;

public class INTEST {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int result=0;
        while (a>0){
            int c = sc.nextInt();
            if(c%b==0){
                result++;
            }
            a--;
        }
        System.out.println(result);
    }
}
